src/Hosting/Hosting/test/WebHostBuilderTests.cs

```diff
@@ -173,7 +173,8 @@ public void ConfigureDefaultServiceProvider(IWebHostBuilder builder)
                     options.ValidateScopes = true;
                 });
 
-            Assert.Throws<InvalidOperationException>(() => hostBuilder.Build().Start());
+            using var host = hostBuilder.Build();
+            Assert.Throws<InvalidOperationException>(() => host.Start());
```

Comment:
Lots 'o C# changes from here on also look odd. Are these also due to merges from 'master'?

Reply:
No this is to prevent a hang. Not disposing generic host builders caused this so I had to add using in several places. @Tratcher is this the issue tacking that bug https://github.com/aspnet/Extensions/issues/1363?

Reply:
Correct